docs.html

```diff
@@ -20,104 +21,195 @@
+          <div class="header-icon"><i class="fab fa-twitter"></i></div>
+        </a>
+      </div>
     </div>
     <div class="nav-close-btn">
       <i class="far fa-times-circle"></i>
     </div>
   </nav>
 
-  <div class="right-side-nav" style="visibility:hidden;">
-    <h1>Contents</h1>
-    <a href="#">
-      <h2 class="nav-input nav-btn">Input</h2>
-    </a>
-  </div>
-
   <!-- floating up btn -->
   <a href="#"><button class="btn floating-btn-md">
       <i class="fas fa-arrow-up"></i></button>
   </a>
 
-  <!-- body -->
+  <div class="main cont-shadow">
+
+    <section id="getting-started" class="pd-2-bot mg-1point4-top">
+      <h1 class="mg-point8-tb">Introduction</h1>
+      <p class="mg-1point4-tb title-md-wt-4">
+        Pabble UI is an open source styling framework, which helps you to quickly build your Dream Project, by focusing
+        much on Functionality, because we take care of your Designs.
+      </p>
+
+      <!-- installation -->
+
+      <h2 id="installation" class="mg-4-top mg-1-bot title-xl-wt-5">Installation</h2>
+      <p class="mg-1point4-tb title-md-wt-4">To use Pabble UI components in your project, Copy and paste the below CSS
+        link in the '&lt;head&gt;' tag of your html, above all other style sheets.</p>
+
+      <script style="height: auto;" type="text/plain" class="language-html code-get-started">
+        <link rel="stylesheet" href="https://pabbleui.netlify.app/Components/components.css"/>
+      </script>
+
+      <!-- font -->
+
+      <h2 id="font" class="mg-4-top mg-1-bot  title-xl-wt-5">Font</h2>
+      <p class="mg-1-tb">
+        For all of our components we have used "Inter" is default font.
+        Still you can use another font of your choice, by placing the new font styling below our CDN in your
+        '&lt;head&gt;' tag. <br><br>
+        CDN for <strong>"Roboto"</strong> font is given below.
+        If you want to use it, just paste it in your <strong>'&lt;head&gt;'</strong> tag.
+      </p>
+
+      <script style="height: auto;" type="text/plain" class="language-html snippet-container">
+        <link
+          rel="stylesheet"
+          href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"/>
+      </script>
 
-  <div class="main-container main ">
+      <!-- font icons -->
 
+      <h2 id="font-icons" class="mg-4-top mg-1-bot title-lg-wt-5">Font icons</h2>
+      <p class="mg-1point4-tb title-md-wt-4">To use the Icon component, you must first add the fontawesome icons. Copy
+        and paste the below link in the '&lt;head&gt;' tag of your html, just after pasting above CSS link and you are
+        done with the installation process!
+        Now you can use any component just by adding their respective class/given code.</p>
 
-    <section id="getting-started" class="pd-1-lr pd-2-bot mg-3-top">
+      <script style="height: 100%" type="text/plain" class="language-html code-get-started">
```

Comment:
Avoid inline styling.

Reply:
These inline are used, only on those snippets that required less height.